### ISSUE TITLE
Laravel Mix not compiling on fresh install

### ISSUE THREAD

**Issue by c-fitzmaurice** [wordplate/wordplate#181]
**Issue:** On a fresh install from composer, Laravel Mix will not compile. Produces the following error: 
```
Error: ENOENT: no such file or directory, open '/Users/Colin/Sites/wordplate/public/themes/wordplate/assets/mix-manifest.json'
```

**Steps to reproduce:**
```
composer create-project wordplate/wordplate
npm install
npm run dev
```

**Comment by vinkla** [comment]
Is the `mix-manifest.json` file in the assets directory?

**Comment by c-fitzmaurice** [comment]
On first run of Mix, it should create the `mix-manifest` if it's not present, no?

**Comment by vinkla** [comment]
Do you get an error when you run `npm dev run`?

**Comment by vinkla** [comment]
I don't have your issue but in my case the script can't run since there are no `assets` directory.

**Comment by c-fitzmaurice** [comment]
Yes, I get the error when I run `npm run dev` on a fresh install of the project, without editing any code.

**Comment by vinkla** [comment]
Okay, then we're having the same issue. Create the `assets` directory in the theme and you'll be good to go. I'll push a fix for this and release a new version. Thanks for reporting the issue!

**Comment by c-fitzmaurice** [comment]
Awesome, thanks for the instant reply. Great project!

**Comment by vinkla** [comment]
This has been fixed in [5.3.1](https://github.com/wordplate/wordplate/releases/tag/5.3.1).
